### ISSUE TITLE
How to get address info from latLng, on Marker>DragEnd

### ISSUE THREAD

**Issue by ghost** [xkjyeah/vue-google-maps#86]

**Comment by danpetban** [comment]
When you call your function in your template like this:
` @dragend="functionName"`

You simply check the passed `$event` object like:
```
functionName (place) {
  console.log('lat: ' + place.latLng.lat)
  console.log('lng: ' + place.latLng.lgn) 
}
```

**Comment by norotico** [comment]
By "address info" he means postal code, street name and such, not the lat/lgn coordinates.

I'm after that too. Does vue-google-maps expose the geocode functionality?

**Comment by xkjyeah** [comment]
Nope, not directly. On drag end, get the new position of the lat long, pass
that into the directions API, and wait for a result.

**Comment by raymayemir** [comment]
@xkjyeah  @band94 why dragend event return data like 
lat: function (){return a}
lng: function (){return b}
???

**Comment by murcoder** [comment]
I've got the same issue:

> lat: function (){return a}

I want to save the coordinates (lang,lat) to the database.
How to get the coordinates?


@raymayemir 
EDIT: Just found out: You have to call it as a funtion! 

```
functionName (place) {
  console.log('lat: ' + place.latLng.lat() )
  console.log('lng: ' + place.latLng.lgn() ) 
}
```
